ui/test/hello_react_test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Yep, updated!  This also led to adding `mocha.opts` which specifies what glob to use when looking for test files.